dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -410,15 +428,14 @@ public void deleteStep(WorkflowStep step) throws DotDataException, AlreadyExistE
 
 	@Override
 	public int getCountContentletsReferencingStep(WorkflowStep step) throws DotDataException{
-		int amount = 0;
+
 		final DotConnect db = new DotConnect();
 
 		// get step related assets
 		db.setSQL(sql.SELECT_COUNT_CONTENTLES_BY_STEP);
 		db.addParam(step.getId());
 		Map<String,Object> res = db.loadObjectResults().get(0);
-		amount=Integer.parseInt(String.valueOf(res.get("count")));
-		return amount;
+		return Integer.parseInt(String.valueOf(res.get("count")));

```

Comment:
replace this thing by ConversionUtils.toInt(res.get("count"), 0);

Reply:
done